### PR TITLE
New NOT_WORKING software list additions

### PR DESCRIPTION
**Pull request by clawgrip**
---------------------------------
sega_beena_cart.xml:
  1-nichi 10-pun de E ga Jouzu ni Kakeru Beena, Anpanman no Waku Waku Game Oekaki, Beena Town e Youkoso, Disney Tanoshii o-Benkyou Series: Mic de Kantan ABC, Disney Tanoshii Oekaki: o-Mise-ya-san o Tsukutchaou!, Cooking Beena: o-Ryouri Dekichatta!, Hello Kitty no Hiragana Katakana o-Namae Kaitemiyou!, Kouchuu Ouja Mushiking: Mori no Tami no Densetsu: Minna de Tanken! Kouchuu no Mori, Partner In TV! o-Uchi ni Wan-chan ga Yattekita, Point Gakushuu Tokei [TeamEurope]

* Fixed copy-paste mistake on 'pokebw'

### CODE REVIEW

File: hash/sega_beena_cart.xml [hunk 1]
```diff
@@ -6,10 +6,63 @@ license:CC0
...
+				<rom name="0 - 0012.png" size="0x1b3ed31" crc="039b0e84" sha1="f90082cea75ec02618deca28bfbae8ec30e2f023"  />
+			</dataarea>
+		</part>
+	</software>
+
 	<software name="asngnk" supported="no">
 		<description>Shoku Iku Series 1 Soreike! Anpanman-Sukikirai Nai Ko Genki na Ko!</description>
 		<year>2005</year>
 		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="食育シリーズ 1 それいけ！アンパンマン すききらいないこ げんきなこ！"/>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:65]: Please remove the spurious spaces from this.

File: hash/sega_beena_cart.xml [hunk 2]
```diff
@@ -31,10 +84,103 @@ license:CC0
...
+				<rom name="0 - 0005.png" size="0x1fbfd2b" crc="4fba7dfa" sha1="bf86eb4bb3f278f425c666b13bf7dd8ecb8ca557" />
+				<rom name="0 - 0006.png" size="0x1be6425" crc="5e439b1e" sha1="dc2e26a93644bd02b6fc4cb803509457863af333" />
+				<rom name="0 - 0007.png" size="0x1cd7f10" crc="140f5dca" sha1="76e5ebf5fa61fc7c0962cb222d9836df1be62d30" />
+				<rom name="0 - 0008.png" size="0x203de88" crc="f8b7cca2" sha1="62be480fe570c4a3724693379fe2917af21c428f" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="dtanoshib" supported="no">
+		<description>Disney Tanoshii o-Benkyou Series: Mic de Kantan ABC</description>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:110]: The honorific “o” is part of the word and should be capitalised if you’re writing in title case.

File: hash/sega_beena_cart.xml [hunk 3]
```diff
@@ -31,10 +84,103 @@ license:CC0
...
+		<year>2005</year>
+		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="ディズニー たのしいおべんきょうシリーズ マイクでかんたんABC♪"/>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:113]: Please remove the spurious spaces.  You’re also missing the 1 (ディズニー たのしいおべんきょうシリーズ① on the box).

File: hash/sega_beena_cart.xml [hunk 4]
```diff
@@ -31,10 +84,103 @@ license:CC0
...
+				<rom name="0 - 0007.png" size="0x1f1492e" crc="06dc678d" sha1="2c9009604acfa517a407951e7a5bff95718ab2ab" />
+				<rom name="0 - 0008.png" size="0x1d92fc9" crc="d3bfc642" sha1="4bcecd682712bd06f6d9901af3ede91efbb35682" />
+				<rom name="0 - 0009.png" size="0x1dc3333" crc="0a59e863" sha1="4ccce518e2b53494c1f68bbc742b398cd6bfa202" />
+				<rom name="0 - 0010.png" size="0x1f05eea" crc="541f1068" sha1="22f50cd233c9059129cb90bf4a7d62b9fa04ff71" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="dtanoshimt" supported="no">
+		<description>Disney Tanoshii Oekaki: o-Mise-ya-san o Tsukutchaou!</description>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:134]: Honorific “o” should be capitalised.

File: hash/sega_beena_cart.xml [hunk 5]
```diff
@@ -31,10 +84,103 @@ license:CC0
...
+				<rom name="0 - 0007.png" size="0x1c22530" crc="f181f7ff" sha1="adfa30fa2de7e3113d6450c2827486c33a20430e" />
+				<rom name="0 - 0008.png" size="0x1fc964e" crc="113420bb" sha1="8bc8a7f1a2660292e9373eeafdde46c6137d208b" />
+				<rom name="0 - 0009.png" size="0x1d0443f" crc="1105baea" sha1="f0869e6ac9165cc9c984519d74c16f2badb1ae06" />
+				<rom name="0 - 0010.png" size="0x204a0d4" crc="2bf18016" sha1="6c7a69cc57224fbb77a77045f64ba5de8f21ae97" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="cookinbe" supported="no">
+		<description>Cooking Beena: o-Ryouri Dekichatta!</description>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:158]: This one, too.

File: hash/sega_beena_cart.xml [hunk 6]
```diff
@@ -60,6 +206,7 @@ license:CC0
 		<description>Soreike! Anpanman Hajimete Kaketa yo! Oboeta yo! Hiragana Katakana - Gojuuon Board Kinou-tsuki</description>
 		<year>2005</year>
 		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="それいけ！アンパンマン どきどき！レスキュードライブ ～カーナビつき～"/>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:209]: That says “Soreike! Anpanman dokidoki! Rescue drive ~Car Navi tsuki~” – doesn’t match the title.

File: hash/sega_beena_cart.xml [hunk 7]
```diff
@@ -81,10 +228,35 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="hkittyhk" supported="no">
+		<description>Hello Kitty no Hiragana Katakana o-Namae Kaitemiyou!</description>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:232]: You know about those “o”

File: hash/sega_beena_cart.xml [hunk 8]
```diff
@@ -152,27 +352,76 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="partnertv" supported="no">
+		<description>Partner In TV! o-Uchi ni Wan-chan ga Yattekita</description>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:356]: This one, too.

File: hash/sega_beena_cart.xml [hunk 9]
```diff
@@ -152,27 +352,76 @@ license:CC0
...
+				<rom name="0 - 0012.png" size="0x1ef088f" crc="47a4b952" sha1="3c5642a7d644fac6d029b183150315ce230fd089" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pointgt" supported="no">
+		<description>Point Gakushuu Tokei</description>
+		<year>2006</year>
+		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="ポイント学習 とけい"/>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:385]: More spurious space.

File: hash/sega_beena_cart.xml [hunk 10]
```diff
@@ -31,10 +84,103 @@ license:CC0
...
+				<rom name="0 - 0005.png" size="0x1e643db" crc="68f58736" sha1="7517f5df1f644cac392150a9e1b1f5bc0aee2130" />
+				<rom name="0 - 0006.png" size="0x1da0ea5" crc="9c567543" sha1="94520e8c1569114ca2b2147f814058073d511df5" />
+				<rom name="0 - 0007.png" size="0x1d9813e" crc="c5bd8898" sha1="b4a717e41959715310c85fab48fc99d5fc88a6e1" />
+				<rom name="0 - 0008.png" size="0x1b3d110" crc="db5ff567" sha1="2d4fcb531a1495630ee4404fbd03e3b7e014fbc9" />
+			</dataarea>
+		</part>
+	</software>
+
 	<software name="frpc" supported="no">
 		<description>Issho ni Henshin Fresh Pretty Cure</description>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:180]: Should probably change this to “PreCure” to match the other ones, unless it actually says “Pretty Cure” in the cover art.

File: hash/sega_beena_cart.xml [hunk 11]
```diff
@@ -206,6 +456,7 @@ license:CC0
 		<description>Yes! PreCure 5 Asonde Oboeyou Hiragana!</description>
 		<year>2007</year>
 		<publisher>Bandai</publisher>
+		<info name="alt_title" value="Yes！プリキュア5 あそんでおぼえよう！ひらがな！"/>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:459]: Any reason you used a halfwidth 5 in the Japanese text?

File: hash/sega_beena_cart.xml (hunk 10 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml:180]: Yes, the cover says "Fresh Pretty Cure"

imagen

File: hash/sega_beena_cart.xml (hunk 11 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: Fixed, thanks!

File: hash/sega_beena_cart.xml (hunk 2 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: Fixed, thanks!

File: hash/sega_beena_cart.xml (hunk 4 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: Fixed, thanks!

File: hash/sega_beena_cart.xml (hunk 7 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: Fixed, thanks!

File: hash/sega_beena_cart.xml (hunk 8 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: Fixed, thanks!

File: hash/sega_beena_cart.xml (hunk 6 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: Fixed, thanks!

File: hash/sega_beena_cart.xml (hunk 3 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: I've added the "1", can you please be more specific about the spaces?

File: hash/sega_beena_cart.xml (hunk 1 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: can you please be more specific about the spaces?

File: hash/sega_beena_cart.xml (hunk 9 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: can you please be more specific about the spaces?

File: hash/sega_beena_cart.xml (hunk 1 above)
Comment (cuavas) [reply hash/sega_beena_cart.xml]: Japanese is written without spaces.  If different parts of the title need to be clearly separated, use appropriate punctuation.  Also, you should definitely be using a fullwidth “1”, not a halfwidth “1” with spaces around it.

File: hash/sega_beena_cart.xml [hunk 12]
```diff
@@ -6,27 +6,172 @@ license:CC0
...
+				<rom name="0 - 0010.png" size="0x1f05eea" crc="541f1068" sha1="22f50cd233c9059129cb90bf4a7d62b9fa04ff71"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="dtanoshimt" supported="no">
+		<description>Disney Tanoshii Oekaki: O-Mise-ya-san o Tsukutchaou!</description>
+		<year>2007</year>
+		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="ディズニーたのしいおえかき おみせやさんをつくっちゃおう!"/>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:137]: Please use fullwidth punctuation in Japanese unless there’s clear evidence that they used a halfwidth form (in this case, the exclamation mark).

File: hash/sega_beena_cart.xml [hunk 13]
```diff
@@ -60,23 +206,48 @@ license:CC0
 		<description>Soreike! Anpanman Hajimete Kaketa yo! Oboeta yo! Hiragana Katakana - Gojuuon Board Kinou-tsuki</description>
 		<year>2005</year>
 		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="それいけ！アンパンマン はじめてかけたよ!おぼえたよ!ひらがな・カタカナ～50音ボード機能つき～"/>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:209]: You’ve got halfwidth exclamation marks here, too.  Also spurious spaces.

File: hash/sega_beena_cart.xml [hunk 14]
```diff
@@ -85,21 +256,48 @@ license:CC0
...
+				<rom name="0 - 0010.png" size="0x01e618a8" crc="571593da" sha1="4125714a58e526b74e7c2e60294bb6450c2bdc8c"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="kouchuu" supported="no">
+		<description>Kouchuu Ouja Mushiking: Mori no Tami no Densetsu: Minna de Tanken! Kouchuu no Mori</description>
+		<year>2005</year>
+		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="甲虫王者ムシキング ～森の民の伝説～ みんなでたんけん!甲虫の森"/>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:283]: This has a halfwidth exclamation mark in it.

File: hash/sega_beena_cart.xml [hunk 15]
```diff
@@ -181,23 +430,24 @@ license:CC0
 		<description>Pocket Monsters Diamond &amp; Pearl Pokemon o Sagase! Meiro de Daibouken!</description>
 		<year>2009</year>
 		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="ポケットモンスター ダイヤモンド・パール ポケモンをさがせ!めいろでだいぼうけん!"/>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:433]: This also has a half-width exclamation mark.

File: hash/sega_beena_cart.xml [hunk 16]
```diff
@@ -206,23 +456,24 @@ license:CC0
 		<description>Yes! PreCure 5 Asonde Oboeyou Hiragana!</description>
 		<year>2007</year>
 		<publisher>Bandai</publisher>
+		<info name="alt_title" value="Yes!プリキュア5 あそんでおぼえよう!ひらがな!"/>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:459]: The two exclamation marks and the “5” in the Japanese part of the title should be fullwidth.

File: hash/sega_beena_cart.xml [hunk 17]
```diff
@@ -256,24 +508,26 @@ license:CC0
 		<description>Samurai Sentai Shinkenger Battle ga Ippai! Iza Mairu!</description>
 		<year>2009</year>
 		<publisher>Bandai</publisher>
+		<info name="alt_title" value="侍戦隊シンケンジャー バトルがいっぱい!いざ参る!"/>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:511]: This also has halfwidth exclamation marks.

File: hash/sega_beena_cart.xml [hunk 18]
```diff
@@ -13,21 +13,21 @@ license:CC0
...
+				<rom name="0 - 0003.png" size="0x197c6c4" crc="30182f89" sha1="9cb9ccee2d21aaf7358ebfca21cb37b332cb5a98"/>
+				<rom name="0 - 0004.png" size="0x1867cca" crc="301d2785" sha1="302902938fe6013d5d3254f8738b07b9494a9856"/>
+				<rom name="0 - 0005.png" size="0x183b750" crc="d61c2830" sha1="fa92264b6f96d49c34c8b06b44469a5e430374f2"/>
+				<rom name="0 - 0006.png" size="0x178d006" crc="be606943" sha1="e28069d2608cb1bb13a3753ba520a0254112e86b"/>
+				<rom name="0 - 0007.png" size="0x191749e" crc="8bfd1ea8" sha1="1a9b78ac4238dfa437285e42f0bc7f8fd4a45e92"/>
+				<rom name="0 - 0008.png" size="0x1756da2" crc="720920cb" sha1="9291de9a4ecbe42ad3eb0fceb03edaf3824de3e9"/>
+				<rom name="0 - 0009.png" size="0x178b5cb" crc="c1c51104" sha1="a2ff91851ebae16649a06474408bea1a2fcb9aba"/>
+				<rom name="0 - 0010.png" size="0x19850b0" crc="520272eb" sha1="5d40da3e50581201c87f5460bbef1c328675a3cc"/>
+				<rom name="0 - 0011.png" size="0x19b8863" crc="77aa6918" sha1="b2095780c92fb55926e8e26759ebc803fe78d677"/>
+				<rom name="0 - 0012.png" size="0x14630ba" crc="d4a4f360" sha1="af125bddb75aba6ffca1a68dd6f226a15cbd7ce6"/>
```
Comment (cuavas) [inline hash/sega_beena_cart.xml:30]: I have no problem with a single space before the `/` in a self-closing element like `<tag attr="val" />` – it’s the spaces in the Japanese titles I object to.  Japanese is written without spaces.

Comment (clawgrip) [reply hash/sega_beena_cart.xml:30]: Yes, but on this XML there was sometimes no spaces, others one and others two. I've nnormalized it to no spaces

File: hash/sega_beena_cart.xml (hunk 17 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: Replaced all halfwidth with regular chars and removed spaces!

File: hash/sega_beena_cart.xml (hunk 15 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: Replaced all halfwidth with regular chars and removed spaces!

File: hash/sega_beena_cart.xml (hunk 16 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: Replaced all halfwidth with regular chars and removed spaces!

File: hash/sega_beena_cart.xml (hunk 13 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: Replaced all halfwidth with regular chars and removed spaces!

File: hash/sega_beena_cart.xml (hunk 14 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: Replaced all halfwidth with regular chars and removed spaces!

File: hash/sega_beena_cart.xml (hunk 12 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: Replaced all halfwidth with regular chars and removed spaces!

File: hash/sega_beena_cart.xml (hunk 9 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: Replaced all halfwidth with regular chars and removed spaces!

File: hash/sega_beena_cart.xml (hunk 3 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: Replaced all halfwidth with regular chars and removed spaces!

File: hash/sega_beena_cart.xml (hunk 1 above)
Comment (clawgrip) [reply hash/sega_beena_cart.xml]: Replaced all halfwidth with regular chars and removed spaces!

**Comment by clawgrip** [comment]
OK, I think all issues are now resolved!

File: hash/sega_beena_cart.xml (hunk 17 above)
Comment (cuavas) [reply hash/sega_beena_cart.xml]: No, that commit replaced fullwidth characters with halfwidth characters.  You’re supposed to use fullwidth punctuation in Japanese text, not halfwidth.

**Comment by cuavas** [comment]
The preferred way is  to use fullwidth punctuation in Japanese, like 。、・！「」  notice how they fit the character grid.  Your latest change replaces all the fullwidth punctuation with halfwidth.

**Comment by clawgrip** [comment]
You're right, I did it the other way. I've now replaced the "!" and "·" with fullwidth punctuation ("！" and "・") in Japanese alt titles

**Pull request merged**